### PR TITLE
MongoDB on ObjectRocket tutorial

### LINKED ISSUE #779: MongoDB on ObjectRocket tutorial

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #779

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-da75a36587/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-47862e2902/).

File: _tutorials/2015-10-23-data-volume-containers.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ For additional assistance, ask the [community](https://community.getcarina.com/)
 
 Learn more about volumes in [Understanding how Carina uses Docker Swarm]({{ site.baseurl }}/docs/concepts/docker-swarm-carina/#volumes).
 
-### Next
+### Next steps
+
+Learn how to [back up and restore container data]({{ site.baseurl }}/docs/tutorials/backup-restore-data/), and [schedule tasks with a cron container]({{ site.baseurl }}/docs/tutorials/schedule-tasks-cron/)
```
Comment (kmsholcomb) [inline _tutorials/2015-10-23-data-volume-containers.md]: End the sentence with a period.

File: _tutorials/2015-10-24-data-stores-mysql.md [hunk 2]
```diff
@@ -189,6 +189,8 @@ For additional assistance, ask the [community](https://community.getcarina.com/)
 
 ### Next steps
 
+Learn how to [back up and restore container data]({{ site.baseurl }}/docs/tutorials/backup-restore-data/), and [schedule tasks with a cron container]({{ site.baseurl }}/docs/tutorials/schedule-tasks-cron/)
```
Comment (kmsholcomb) [inline _tutorials/2015-10-24-data-stores-mysql.md]: End the sentence with a period.

File: _tutorials/2015-10-25-data-stores-mongodb.md [hunk 3]
```diff
@@ -164,7 +164,9 @@ For additional assistance, ask the [community](https://community.getcarina.com/)
 * [Getting Started with the mongo Shell](http://docs.mongodb.org/master/tutorial/getting-started-with-the-mongo-shell/)
 * [Use overlay networks in Carina]({{ site.baseurl }}/docs/tutorials/overlay-networks/)
 
-### Next
+### Next steps
+
+Learn how to [back up and restore container data]({{ site.baseurl }}/docs/tutorials/backup-restore-data/), and [schedule tasks with a cron container]({{ site.baseurl }}/docs/tutorials/schedule-tasks-cron/)
```
Comment (kmsholcomb) [inline _tutorials/2015-10-25-data-stores-mongodb.md]: End the sentence with a period.

File: _tutorials/2015-10-25-data-stores-redis.md [hunk 4]
```diff
@@ -107,6 +107,8 @@ For additional assistance, ask the [community](https://community.getcarina.com/)
 
 ### Next steps
 
+Learn how to [back up and restore container data]({{ site.baseurl }}/docs/tutorials/backup-restore-data/), and [schedule tasks with a cron container]({{ site.baseurl }}/docs/tutorials/schedule-tasks-cron/)
```
Comment (kmsholcomb) [inline _tutorials/2015-10-25-data-stores-redis.md]: End the sentence with a period.

**Comment by kmsholcomb** [comment]
Hi @everett-toews: I've attached docs for two articles. One Word doc (and its PDF copy) has just a few nitpicky edits for the "Back up and restore container data" article, and the other Word doc (and its PDF copy) has my suggested edits for the "Connect a Carina container to an ObjectRocket MongoDB instance" article. Additionally, I left a comment in the the other files on the **Files changed** tab. Please let me know if you have any questions. Thanks!

[Back up and restore container data_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/215855/Back.up.and.restore.container.data_editsKH.pdf)
[Back up and restore container data_editsKH.docx](https://github.com/getcarina/getcarina.com/files/215856/Back.up.and.restore.container.data_editsKH.docx)
[Connect a Carina container to an ObjectRocket MongoDB instance_editsKH.pdf](https://github.com/getcarina/getcarina.com/files/215854/Connect.a.Carina.container.to.an.ObjectRocket.MongoDB.instance_editsKH.pdf)
[Connect a Carina container to an ObjectRocket MongoDB instance_editsKH.docx](https://github.com/getcarina/getcarina.com/files/215853/Connect.a.Carina.container.to.an.ObjectRocket.MongoDB.instance_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-60bde19e02/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-f4da41f101/)

**Pull request merged**